### DISCUSSION TITLE
DataPicker 디자인 문의

### DISCUSSION THREAD

**Discussion by KIMSEI1124** [What-s-Your-Plan/wypl-frontend#28] [General]
현재 DataPicker 의 오늘로 가는 버튼의 디자인이 고민이 됩니다!
영어로 된 단어나 한글로 된 단어중 어떤 것을 선택할지 고민이 되는데 보시고 알려주시면 감사할 것 같습니다!

| **한글** | **영어** |
| --- | --- |
|![2024-09-20_14-02-20](https://github.com/user-attachments/assets/86880964-18a9-48e9-b11a-3c6a46ec47dc) | ![2024-09-20_14-02-29](https://github.com/user-attachments/assets/5162cf18-feb1-4d37-b878-1f7d9ba64a51) |

**Comment by KIMSEI1124** [comment]
@jiwon83 @sjhjack 여유 되실때 한번 보고 천천히 알려주시면 됩니다!

**Comment by sjhjack** [comment]
저는 한글이 좋아보여요.
요일이 한글로 돼있어서 영어는 뜬금없다고 느껴져요.
강조할만큼 중요한 버튼도 아니라고 생각해요.

**Reply by KIMSEI1124** [reply]
그럼 혹시 `오늘` 버튼이 화살표 사이에 있는건 어떻게 생각하시나요?

**Reply by sjhjack** [reply]
별로일 것 같아요.
사용자가 반대 화살표 누르려다가 `오늘` 버튼을 잘못 누른다면 불편할 것 같아요.
개인적으로 디자인도 별로라고 생각하구요.

아까 생각하면서 화살표 위 자리도 고민해 봤는데, 버튼 세 개가 모여있으면 난잡할 것 같더라구요..!
